earth_enterprise/rpms/opengee-common/pre-uninstall.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
```

Comment:
I wonder if this really should be a post-uninstall rather than a pre-uninstall.  Things still exist under the group ownership until the actual remove is executed...

Reply:
Yeah. The only issue used to be, if the script was trying to access `/etc/init.d/gevars.sh` which wouldn't exist in the post-uninstall. However, I moved `gevars.sh` to the `common` package, so that shouldn't be a limitation here.

I have not fixed the Fusion install scripts, yet.  There also needs to be the `gefusionuser` creation and removal in them.